### DISCUSSION TITLE
Schedules don't work, what am I doing wrong?

### DISCUSSION THREAD

**Discussion by phaelium** [tagyoureit/nodejs-poolController#1033] [Nixie]
So, I've rebuilt this thing from scratch trying to simplify it as I'm having a heck of a time getting schedules to work - at all.  I don't have a controller so I'm using the Nixie Stand Alone, and my setup is pretty basic.  I'm not using RS485 so instead I'm using 4 relays to run the different pump speeds via REM.  When I manually turn on the pool then I have the low speed running on the pump via a relay, and if I setup the other speeds as features then they all work as expected.  

However, I can't get any schedules to work.  They never start, doesn't seem to matter what circuit I pick or what time they just don't do anything.  And I can't find anything in the logs that even reference schedules running.  

I'm guessing something is wrong with either my config or perhaps in NPM that's causing the software to not even evaluate the schedules?  I'm running 18.20 but I've tried 20 as well, didn't make any difference.  RPI 4B with the pm2 setup from the wiki.

Please help!  :)  

[replay (4).zip](https://github.com/user-attachments/files/17159545/replay.4.zip)

**Comment by bruno-monk** [comment]
I’m going to add that my schedules have stopped firing as well, even with pulling new packages. I can manually start my pump but schedule won’t fire.

**Comment by MichelMoriniaux** [comment]
I updated controller dashboard and relay manager this weekend from a 1 year old version, schedules are not firing anymore.
everything else seems to work

**Comment by tagyoureit** [comment]
Pushed a fix for this.

**Reply by bruno-monk** [reply]
Thank you: schedules are now firing.
